api/src/main/java/io/opentelemetry/api/baggage/Baggage.java

```diff
@@ -39,6 +39,23 @@ static Baggage current() {
     return BaggageUtils.getCurrentBaggage();
   }
 
+  /**
+   * Returns the {@link Baggage} from the specified {@link Context}, falling back to a empty {@link
+   * Baggage} if there is no baggage in the context.
+   */
+  static Baggage fromContext(Context context) {
+    return BaggageUtils.getBaggage(context);
+  }
+
+  /**
+   * Returns the {@link Baggage} from the specified {@link Context}, or {@code null} if there is no
+   * baggage in the context.
+   */
+  @Nullable
+  static Baggage fromContextOrNull(Context context) {
```

Comment:
Do we need this? People can compare with empty or can check for size of entries.

Reply:
May be in a separate PR.

Reply:
+1